src/ansys/mapdl/core/cli/__init__.py

```diff
@@ -48,6 +49,12 @@ def main(ctx):
     main.add_command(stop)
     main.add_command(list_instances, name="list")
 
+    # HPC commands
+    # pymapdl hpc submit
+    # pymapdl hpc list
+    # pymapdl hpc stop
+    main.add_command(submit)
+
     def old_pymapdl_convert_script_entry_point():
         print(
             """This CLI function has been deprecated. Please use instead:
```

Comment:
```suggestion
            """This CLI function has been deprecated. Use the following instead:
```

Reply:
Later, "please" is also used, but it is considered unnecessary in technical documentation.